code/src/pages/product-details/ProductDetailsPage.tsx

```
@@ -8,17 +9,25 @@ import useErrorPageRedirect from "@/hooks/use-error-page-redirect/useErrorPageRe
 import { productNotFoundRedirectConfig } from "@/pages/product-details/ProductsDetailsPage.constants";
 import ProductDetailsContainer from "@/pages/product-details/components/product-details-container/ProductDetailsContainer";
+import { useUpdateViewedProductsMutation } from "@/store/api/viewHistoryApi";
+import { useUpdateViewedProductsMutation } from "@/store/api/viewHistoryApi";
 
 import "@/pages/product-details/ProductDetailsPage.scss";
 
 const ProductDetailsPage = () => {
   const { productId } = useParams<ProductDetailsPageParams>();
   const { renderRedirectComponent } = useErrorPageRedirect();
+  const [addViewProduct] = useUpdateViewedProductsMutation();
+  const [addViewProduct] = useUpdateViewedProductsMutation();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'addViewProduct'. Consider to delete it or rename it.

'addViewProduct' is defined here:


(lint/suspicious/noRedeclare)